source/help/policies/content-types.md

```diff
@@ -21,10 +22,10 @@ All submissions to arXiv must meet the standards for [moderation and scholarly i
 - Undergraduate or high school research
 - Very short work
```

Comment:
add: books whose target audience are below graduate level? we've declined those in the past with the reason that they're not of value to professional researchers.

Reply:
I'm hesitant to add this initially as the requirement for the scholarly level of research in each submission is covered by the moderation page. Lets circle back to the EAC if we see confusion about this from users in the future.